test/transfer_rule/transfer_from.js

```diff
@@ -0,0 +1,93 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const web3 = require('../test_lib/web3.js');
+const utils = require('../test_lib/utils.js');
+const TokenRules = artifacts.require('TokenRules');
+const TransferRule = artifacts.require('TransferRule');
+const EIP20TokenMock = artifacts.require('EIP20TokenMock');
```

Comment:
Let's be consistent with 
artifacts.require('TokenRules') => artifacts.require('TokenRules.sol')
artifacts.require('TransferRule'); => artifacts.require('TransferRule.sol')

Reply:
not a hard requirement.